cap-fs-ext/src/dir_ext.rs

```diff
@@ -157,6 +169,18 @@ pub trait DirExtUtf8 {
     fn open_dir_nofollow<P: AsRef<str>>(&self, path: P) -> io::Result<Self>
     where
         Self: Sized;
+
+    /// Removes a file or symlink from a filesystem.
+    ///
+    /// Removal of symlinks has different behavior under Windows - if a symlink points
+    /// to a directory, it cannot be removed with the remove_file operation. This
+    /// method will remove files and all sy mlinks
```

Comment:
Here and above, typo: "sy mlinks". Also, please add a period at the end of the sentence.

Reply:
e6cb016